umbraco-cloud/security.md

```diff
@@ -125,9 +125,9 @@ Since December 8th, 2020 all Umbraco Cloud sites uses Cloudflare for DNS, so new
 If you are unsure whether your Cloud project uses Cloudflare or not, get in touch with the friendly support team, and they will help you out.
 {% endhint %}
 
-**Reverse Proxy version (eg. Cloudflare)**
+**Default setup (reverse Proxy version, eg. Cloudflare)**
 
-When using Cloudflare, which is the default setup for all Cloud projects, the project will from behind a reverse proxy get the IPs from the `CF-Connecting-IP` header. In this case, which is most cases, use the first variation here to restrict access to your backoffice using IP filtering.
+For all cloud projects since December 8th, 2020, Cloudflare is used as a reverse proxy service for your project. This means that you'll need to get the IPs from the `X-Forwarded-For` header. Use the first variation here to restrict access to your backoffice using IP filtering.
```

Comment:
"Use the first variation **here** to ..."

What is the "here" referring to?
I realize this is not something you've added specifically - but if it's referring to the code snippet a few lines below, should we ensure that the code snippet comes directly after this line? 🤔